scripts/migrate-yaml.py

```diff
@@ -16,55 +16,26 @@
 # (c) 2016, Matthew Thode <matt.thode@rackspace.com>
 
 """
-Print out values that have changed, by default just additions and changes.
-
-Additions can be taken as is and placed into an overrides file, but should
-you should verify you still want to keep it.
-
-Removals are not printed by default because not having an override is the
-same as taking a default.
-
-Changes are printed, showing what you overrode and what is getting overridden.
-
-CAUTION: danger-mode is only if you KNOW what you are overriding doesn't
-    need to change, warrantee is void if used.
+Write a file containing new overrides with the new defaults stripped from the
+old overrides and conflicts listed in their own keys.
 """
 
 from __future__ import print_function
 import argparse
-from collections import namedtuple
-from deepdiff import DeepDiff
 import yaml
 
 
 def parse_args():
     parser = argparse.ArgumentParser()
     parser.add_argument('--overrides', dest='overrides', required=True,
                         help='The path to the file with overrides.')
-    parser.add_argument('--info', dest='info', type=bool, default=False,
-                        help='Shows what is NOT overridden.')
-    parser.add_argument('--danger-mode', dest='danger',
-                        type=bool, default=False,
-                        help='Print output usable as an overrides file.')
+    parser.add_argument('--defaults', dest='defaults', required=True,
+                        help='The path to the file with defaults.')
+    parser.add_argument('--output-file', dest='output_file', required=True,
```

Comment:
argparse allows you to define this type as a File so you get an open file handler